install/Makefile

```diff
@@ -141,6 +142,22 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export FIGURINE_VERSION ?= 0.2.2
+## Install figurine foir printing your name in style (darwin)
+figurine-darwin:
+	$(CURL) -o - https://github.com/arsham/figurine/releases/download/v$(FIGURINE_VERSION)/figurine_darwin_v$(FIGURINE_VERSION).tar.gz | tar -zxO  figurine > $(INSTALL_PATH)/figurine
+	chmod +x $(INSTALL_PATH)/figurine
+
+## Install figurine foir printing your name in style (linux)
+figurine-linux:
```

Comment:
There is no need for figurine to break it into 2 targets as the process for linux and OSX is the same. Replace the `darwin` and `linux` keywords with `$(OS)` (see other targets for an example) and consolidate into a single target